dev-server/oidc/Dockerfile

```
@@ -0,0 +1,16 @@
+FROM node:10
```

```suggestion
FROM node:lts-slim
```
Even if it's only used in a CI environment, try to use the smallest possible image. It does take longer to download 350MB image over of a 50MB image.
It's also preferable to use a LTS release unless there are uncontrollable requirements that forces the use of an older image.